### ISSUE TITLE
Webhooks are working properly but I'm not getting messages on my smartphone

### ISSUE THREAD

**Issue by marcis** [daveebbelaar/airbnb-whatsapp-bot#24]
Hi guys,

I followed the instructions until point 3.

![image](https://github.com/user-attachments/assets/cd05f038-655d-474d-ad64-4aea2392297a)

On "Integrate WhatsApp" section, I got a call to my local webhook (through ngrok) when clicking on "Test" at FB developers site and received "THIS IS A TEXT MESSAGE" on my personal WhatsApp account. Perfect!

![image](https://github.com/user-attachments/assets/bc6a18e7-f240-4de1-8151-e73aa6f0df45)

Then I replied to that message but got no webhook nor, of course, reply.

What am I missing, please?

**Comment by qrzeller** [comment]
I got the same issue. I didn't click "subscribe" for the messages section in their webhooks settings.

**Comment by oscarnevarezleal** [comment]
Same here :/

**Comment by Dan-White0** [comment]
I was running into this issue. I had the subscription for messages ticked, and when I pressed test it all worked, but I would not get a reply to any messages I sent. 
I tried the answer from [this](https://stackoverflow.com/questions/79175537/whatsapp-business-api-messages-webhook-not-triggering-even-with-manual-testin) stack overflow which seems to have fixed it!

tldr:
```bash
curl -X POST \
'https://graph.facebook.com/v21.0/WHATSAPP_BUSINESS_ACCOUNT_ID/subscribed_apps' \
-H 'Authorization: Bearer ACCESS_TOKEN'
```

**Comment by crmejia** [comment]
> I was running into this issue. I had the subscription for messages ticked, and when I pressed test it all worked, but I would not get a reply to any messages I sent. I tried the answer from [this](https://stackoverflow.com/questions/79175537/whatsapp-business-api-messages-webhook-not-triggering-even-with-manual-testin) stack overflow which seems to have fixed it!
> 
> tldr:
> 
> curl -X POST \
> 'https://graph.facebook.com/v21.0/WHATSAPP_BUSINESS_ACCOUNT_ID/subscribed_apps' \
> -H 'Authorization: Bearer ACCESS_TOKEN'

This worked for me. Two things that I had other things that I had to do:
1. My Access Token had expired after only a few hours.
2. Make sure to use the correct API version. At the time of writing its v22.0.
